src/Symfony/Bundle/FrameworkBundle/Command/KernelDebugCommand.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Component\Console\Helper\Helper;
+use Symfony\Component\Console\Helper\TableSeparator;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\HttpKernel\Kernel;
+use Symfony\Component\HttpKernel\KernelInterface;
+
+/**
+ * A console command to display kernel information.
+ *
+ * @author Roland Franssen <franssen.roland@gmail.com>
+ */
+class KernelDebugCommand extends ContainerAwareCommand
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('debug:kernel')
+            ->setDescription('Displays kernel information')
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        /** @var $kernel KernelInterface */
+        $kernel = $this->getContainer()->get('kernel');
+        $baseDir = realpath($kernel->getRootDir().DIRECTORY_SEPARATOR.'..');
+        $bundles = array_map(function ($bundle) use ($baseDir) {
+            return '* '.$bundle->getName().' (<comment>'.$this->formatPath($bundle->getPath(), $baseDir).'</>)';
+        }, $kernel->getBundles());
+        sort($bundles);
+
+        $io->title('Welcome to Symfony '.Kernel::VERSION);
```

Comment:
I'd remove these `$io->title('Welcome to...')` and `$io->success('Your app is ready...')` calls. They mimic the "Welcome Page" HTML contents, but here they look out of place. And they don't provide any useful information compared to the tables below.